### DISCUSSION TITLE
why the score value in the returned search results is ascending order [{score:0.46}, {score: 0.47}]

### DISCUSSION THREAD

**Discussion by grimmerk** [milvus-io/milvus#27697] [Q&A and General discussion] [answered]
Either using `topK` to get one search result or using `limit/offset` to get two related search results, the score value in each element in the returned response’s results is in ascending order. E.g. `[{score:0.46}, {score: 0.47}]`. I expect this score field to be about dot/cosine similarity, and the order should be descending. e.g. `[{score: 0.9}, {score: 0.8}]`. Is there any misunderstanding?

**Comment by xiaofan-luan** [comment, marked as answer]
using L2 metrics, smaller means similar

**Reply by xiaofan-luan** [reply]
dot and cosine, large means similar

**Reply by grimmerk** [reply]
Thank you very much, and I'm using L2 metrics.
